checker/check.go

```diff
@@ -339,6 +358,7 @@ func (triggerChecker *TriggerChecker) checkTargets(
 	if err != nil {
 		return lastState, needToDeleteMetric, err
 	}
+	logger.Debug().Msg(fmt.Sprintf("Last state: %v", lastState))
```

Comment:
is this need?

if needs i think better will be: 

logger.Debug().String('last_state', lastState).Msg("some text")